src/pds/registry/utils/treks/create_treks_pds4.py

```diff
@@ -0,0 +1,166 @@
+"""Script to build pds4 xml for Treks OGC WMTS layers.
+
+The layers are scraped using the Treks API.
+"""
+import argparse
+import logging
+import os
+import time
+from pathlib import Path
+
+import requests
+
+from .product_collection_builder import create_collection_pds4
+from .product_service_builder import ProductServiceBuilder
+
+
+def main():
+    """Generate PDS4 XML labels for Treks OGC/WMTS GIS Service."""
+    # valid targets for treks
+    valid_targets = [
+        # planets
+        "mars",
+        "mercury",
+        "venus",
+        # moons
+        "moon",
+        "phobos",
+        "titan",
+        # asteroids
+        "ceres",
+        "ryugu",
+        "vesta",
+        # run script on all targets
+        "all"
+    ]
+
+    # set up command line args
+    parser = argparse.ArgumentParser(description="Create and save pds4 xml labels created for Treks",
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument("-s",
+                        "--save-xml",
+                        action="store_true",
+                        help="True if you want the xml files to be saved",
+                        default=True)
+    parser.add_argument("-d",
+                        "--destination-directory",
+                        help="Directory to save pds4 xml files",
+                        default="treks_xml")
+    parser.add_argument("-v",
+                        "--verbose",
+                        action="store_true",
+                        help="Verbosity of file creation",
+                        default=False)
+    parser.add_argument("-l",
+                        "--save-logs",
+                        action="store_true",
+                        help="Save logs of run",
+                        default=False)
+    parser.add_argument("-t",
+                        "--target",
+                        help=f"Treks target to generate labels for: {valid_targets}",
+                        default="titan")
+
+    args = parser.parse_args()
+
+    # parse args
+    save_xml = args.save_xml
+    dest = args.destination_directory
+    verbose = args.verbose
+    save_logs = args.save_logs
+    target = args.target.lower()
+
+    # remove whitespace from target
+    target = target.strip()
+    target = target.replace(" ", "")
+
+    selected_targets = []
+    if target == 'all':
+        selected_targets = valid_targets[:-1]
+    else:
+        selected_targets = [target]
+
+    # initialize logger
+    if save_logs:
+        logging.basicConfig(filename='create-treks-pds4-log.log', level=logging.WARNING)
+        logging.warning("target,product_label,missing_data_location,tag")
+
+    total_start = time.time()
+    for target in selected_targets:
+        target_dest = dest + "/" + target
+
+        if verbose:
+            print("Creating pds4 xml for target:", target)
```

Comment:
Using both `logging` and `print` statements … why not just stick with `logging` exclusively?

Reply:
I can stick to logging exclusively if that's best practices. I already had print in there, and I just added logging on top since I didn't know if I should have gotten rid of it or not.